pontoon/base/views.py

```diff
@@ -377,6 +377,44 @@ def get_translations_from_other_locales(request):
     return JsonResponse(payload, safe=False)
 
 
+@utils.require_AJAX
+def get_sibling_entities(request):
+    """Get entities preceding and succeeding the current entity"""
+    try:
+        entity = int(request.GET["entity"])
+        locale = request.GET["locale"]
+    except (MultiValueDictKeyError, ValueError) as e:
+        return JsonResponse(
+            {"status": False, "message": "Bad Request: {error}".format(error=e)},
+            status=400,
+        )
+
+    entity = get_object_or_404(Entity, pk=entity)
+    locale = get_object_or_404(Locale, code=locale)
+    preferred_source_locale = ""
```

Comment:
Nit: I'd add a blank line above this line to visually separate the blocks.

Reply:
Thank you, I'll keep it in mind when I'm working. Where did you learn your best practice methods when writing code.

Reply:
I wouldn't call this a best practice, just a personal preference. :)

Ideally, we should just leave it for the linters to do their job and not talk about code style during reviews. So I'd be totally fine if you ignore my comment. :)